packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/QuickActionsPluginTests.swift

```diff
@@ -58,80 +55,30 @@ class QuickActionsPluginTests: XCTestCase {
       icon: UIApplicationShortcutIcon(templateImageName: "search_the_thing.png"),
       userInfo: nil)
 
-    let call = FlutterMethodCall(methodName: "clearShortcutItems", arguments: nil)
-    let mockChannel = MockMethodChannel()
-    let mockShortcutItemProvider = MockShortcutItemProvider()
-    let mockShortcutItemParser = MockShortcutItemParser()
-
-    mockShortcutItemProvider.shortcutItems = [item]
-
-    let plugin = QuickActionsPlugin(
-      channel: mockChannel,
-      shortcutItemProvider: mockShortcutItemProvider,
-      shortcutItemParser: mockShortcutItemParser)
-
-    let resultExpectation = expectation(description: "result block must be called.")
-    plugin.handle(call) { result in
-      XCTAssertNil(result, "result block must be called with nil.")
-      resultExpectation.fulfill()
-    }
-
-    XCTAssertEqual(mockShortcutItemProvider.shortcutItems, [], "Must clear shortcut items.")
-    waitForExpectations(timeout: 1)
-  }
-
-  func testHandleMethodCall_getLaunchAction() {
-    let call = FlutterMethodCall(methodName: "getLaunchAction", arguments: nil)
-
-    let mockChannel = MockMethodChannel()
+    let mockMessenger = MockBinaryMessenger()
     let mockShortcutItemProvider = MockShortcutItemProvider()
     let mockShortcutItemParser = MockShortcutItemParser()
 
     let plugin = QuickActionsPlugin(
-      channel: mockChannel,
+      messenger: mockMessenger,
       shortcutItemProvider: mockShortcutItemProvider,
       shortcutItemParser: mockShortcutItemParser)
 
-    let resultExpectation = expectation(description: "result block must be called.")
-    plugin.handle(call) { result in
-      XCTAssertNil(result, "result block must be called with nil.")
-      resultExpectation.fulfill()
-    }
-
-    waitForExpectations(timeout: 1)
-  }
-
-  func testHandleMethodCall_nonExistMethods() {
```

Comment:
why are we removing these tests? can you run a coverage report in xcode? IIRC it used to be 100%

Reply:
You can't call a method that doesn't exist via Pigeon APIs, so the test is no longer meaningful.